### DISCUSSION TITLE
EU (DE) Stores - ADD Links for NVIDIA RTX 4090 PLS

### DISCUSSION THREAD

**Discussion by w3bprinz** [jef/streetmerchant#3169] [General]
Could you please add links for the 4090 and German EU/DE Stores?

Here are some links for the MSI 4090 Suprim X

Mindfactory:
https://www.mindfactory.de/product_info.php/24GB-MSI-GeForce-RTX-4090-Suprim-X-Aktiv-PCIe-4-0-x16--Retail-_1469833.html

Caseking:
https://www.caseking.de/msi-geforce-rtx-4090-suprim-x-24g-24576-mb-gddr6x-gcmc-331.html

Amazon:
https://www.amazon.de/MSI-GeForce-4090-SUPRIM-Grafikkarte/dp/B09YCBDG6X?ref_=ast_sto_dp&th=1&psc=1

Proshop:
https://www.proshop.de/Grafikkarte/MSI-GeForce-RTX-4090-SUPRIM-X-24GB-GDDR6X-RAM-Grafikkarte

Cyberport:
https://www.cyberport.de/?DEEP=2E12-3NG

If you tell me how I can contribute to your project I would do it on my own and will send updated store files.

**Comment by Mr-Tech-13** [comment]
**How do i add links to my new *store.ts* file?**
Copy and paste this into your new *store.ts* file. Be sure to replace everythong with the details relevant to you. IE: replace things like *(copied selector)* or anything blank such as *brand: 'new brand',* Any issues send a message in <#776482351102427196> or <#773939292547973130> 

import {Store} from './store';

export const (store): Store = {
currency: '(can be anything in https://github.com/jef/streetmerchant/blob/main/src/store/model/store.ts#L313)',
labels: {
inStock: {
container:
'(copied selector)',
text: ['(add to cart text)'],
},
outOfStock: {
container:
'(out of stock selector. Can be anything that says out of stock)',
text: ['(out of stock text)'],
},
},
links: [
{
brand: 'test:brand',
model: 'test:model',
series: 'test:series',
url: 'https://www.power.no/data-og-tilbehoer/p-1128968/',
},
// VV Copy and paste as many times as needed but don't leave this one blank! VV
{
brand: '',
model: '',
series: '',
url: '',
},


**How do i add the 3050/3090ti**
https://www.jef.buzz/streetmerchant/help/general/#adding-a-store and https://discord.com/channels/773913070665859073/776482351102427196/933750485222060132
That should get you started. This is from the street merchant discord
@

**Comment by w3bprinz** [comment]
Thanks, I already did and submitted a pull request.
